draft outline for all episodes (#2)
carpentries/lessondev-trainer-training: episodes/backwards-design.md

---
title: 'Backwards Lesson Design'
teaching: 30
exercises: 30
---

:::::::::::::::::::::::::::::::::::::: questions 

- What is meant by _backwards design_ of curriculum?
- What are the main steps of the backwards design process?
- What are the advantages and disadvantages of backwards design?

::::::::::::::::::::::::::::::::::::::::::::::::

::::::::::::::::::::::::::::::::::::: objectives

By the end of this session, participants should be able to...

- discuss the application of backwards design to curriculum for short-format training events
  like Carpentries workshops.
- list at least three advantages and at least one disadvantage of 
  taking a backwards design approach to lesson/curriculum design.

::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::::::::::::: checklist

## Reading List

- FIXME

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::::::::::::: discussion

## Introductions

The first discussion session will begin with a full round of Introductions,

> where each trainee will be asked to share some details about themselves

Reviewer: folks leading this session should probably introduce themselves as well?

Author: ```suggestion
where trainees and facilitators will be given the opportunity to share some details about themselves
```
